adapters/pubmatic/pubmatic.go

```diff
@@ -337,11 +342,17 @@ func (a *PubmaticAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder
 func (a *PubmaticAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
 	errs := make([]error, 0, len(request.Imp))
 
-	wrapExt := ""
+	wrapperExt := new(pubmaticWrapperExt)
```

Comment:
Nitpick: It's idiomatic in Go to use `&pubmaticWrapperExt{}`.

Reply:
Fixed.